auth.go

```diff
@@ -158,7 +162,8 @@ func (a *Auth) CheckPassphrase(passphrase string) error {
 }
 
 // Op sets a public key as a known operator.
-func (a *Auth) Op(key ssh.PublicKey, d time.Duration) {
+// comment is the string that follows the public key in openssh authorized_keys format
+func (a *Auth) Op(key ssh.PublicKey, comment string, d time.Duration) {
```

Comment:
Can we avoid changing `Op(...)` and `Allow(...)` by introducing a new `Reserve(...)` function?

Reply:
If you want to keep this a half-way solution, could maintain the same internal storage as a reconstructed "comment" for now, but I would appreciate if can at least converge on a "final" public code interface. :)